types/check-types/index.d.ts

```diff
@@ -0,0 +1,166 @@
+// Type definitions for check-types 7.3
+// Project: https://github.com/philbooth/check-types.js
+// Definitions by: idchlife <https://github.com/idchlife>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/
+// TypeScript Version: 2.2
+
+/**
```

Comment:
No need to include this comment... that's why we have the header.

Reply:
unfortunately typescript 2.1 check fails since there is no object type. how can it be resolved to define typescript version 2.2?

about comment with definitions - I just copied them from other types in this repository. definitions in comment can be removed, yes?

Reply:
I meant the comment from line 7 - 10